workers/unity/Packages/io.improbable.gdk.gameobjectcreation/.codegen/Source/GameObjectGenerationJob.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Improbable.Gdk.CodeGeneration.FileHandling;
+using Improbable.Gdk.CodeGeneration.Jobs;
+using Improbable.Gdk.CodeGeneration.Model.Details;
+using Improbable.Gdk.CodeGeneration.Utils;
+
+namespace Improbable.Gdk.CodeGenerator.GameObjectCreation
```

Comment:
I think we should rename this module since it encompasses everything to do with game objects (not just creation)

Reply:
Agreed, though out of scope for this